tests/unit/UI/Badges/SimpleBadge.spec.js

```diff
@@ -0,0 +1,26 @@
+import { mount } from "@vue/test-utils";
+import SimpleBadge from "@/components/UI/Badges/SimpleBadge";
+
+describe("SimpleBadge.vue", () => {
+  it("renders title correctly", () => {
```

Comment:
if doing tests on things like title, description already covers the `truthy` part, we can remove the truthy checks from the other files too right?

Reply:
Not sure if I understood your question. Can you elaborate?